tools/server/webui/src/utils/types.ts

```diff
@@ -89,3 +112,12 @@ export interface CanvasPyInterpreter {
 }
 
 export type CanvasData = CanvasPyInterpreter;
+
+// a non-complete list of props, only contains the ones we need
+export interface LlamaCppServerProps {
+  build_info: string;
+  model_path: string;
+  n_ctx: number;
+  has_multimodal: boolean;
+  // TODO: support params
```

Comment:
In a next PR, we can use the server's default `params` to override the settings, this will address https://github.com/ggml-org/llama.cpp/issues/13277